### PR TITLE
fix folder structure and thunk for watchlist

### PR DESCRIPTION
**Pull request by hondahmd**
- folder structrrrrrrrrrrrrrrrrue

### CODE REVIEW

File: frontend-opal/src/reducers/Watchlists.js [hunk 1]
```diff
@@ -1,5 +1,7 @@
 const pairs = {
```
Comment (adamgyulavari) [inline frontend-opal/src/reducers/Watchlists.js]: @hondahmd name of the file should start with capital if it's a class or component, this is neither

File: frontend-opal/tests/PortfolioContainer/Allocation.test.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import renderer from 'react-test-renderer';
 
-import AllocationInfo from '../src/components/PortfolioContainer/AllocationInfo';
+import AllocationInfo from '../../src/components/PortfolioContainer/AllocationInfo';
```
Comment (adamgyulavari) [inline frontend-opal/tests/PortfolioContainer/Allocation.test.js]: @hondahmd cant use the import root as configured? `from 'components/...'`

**Comment by adamgyulavari** [comment]
also sth and put some files is unnecessary, you can just name this `fix folder structure and thunk for watchlist`

**Comment by hondahmd** [comment]
@adamgyulavari All comments resolved. Naming rules for other parts will be discussed in the future. The alias path can now be used for test files. I'll pay attention to the title of PR :)

**Comment by adamgyulavari** [comment]
but you haven't changed the name of the PR :)

**Pull request merged**